### PR TITLE
Supporting .net standard 1.6.

### PR DESCRIPTION
**Pull request by iremmats**
New major version. 
.NET standard 2.0.
People have to upgrade their Visual Studio... :)

### CODE REVIEW

**Comment by marbergq** [comment]
Does not build, i don't think the FormatterServices are included in the netstandard, thus needs an nuget dependency (?)

**Comment by iremmats** [comment]
"It works on my machine..." :)

Think it's AppVeyor that doesn't support it yet.

See https://andrewlock.net/building-asp-net-core-2-0-preview-2-packages-on-appveyor/. Might not be 100% relevant.

**Comment by marbergq** [comment]
FormatterServices.GetUninitializedObject in netstandard2.0 is a wrapper for          RuntimeHelpers.GetUninitializedObject(type); 

https://github.com/dotnet/corefx/blob/6c8411036d4b196c48433ac5e0118b2de525c01c/src/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/FormatterServices.cs#L185 

Does that exist in 1.6?

**Comment by iremmats** [comment]
Cleaned two packages that weren't used and changed back to .net standards 1.6 AND net46. Is it OK to not support 4.5?
The line that caused all the "problems". In which scenario is it useful?
https://github.com/collector-bank/common-restcontracts/blob/94072281aa9482bff62453e2e69c74008c015d2d/src/Collector.Common.RestContracts/RequestBase.cs#L45

File: src/Collector.Common.RestContracts/RuntimeHelper.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (marbergq) [inline src/Collector.Common.RestContracts/RuntimeHelper.cs]: unnecessary using

File: src/Collector.Common.RestContracts/RuntimeHelper.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using System.Text;
+using System.Reflection;
+
+namespace Collector.Common.RestContracts
+{
+    using System.Reflection;
+
+    public class RuntimeHelper
+    {
+        private static readonly Func<Type, object> getUninitializedObjectDelegate =
```
Comment (marbergq) [inline src/Collector.Common.RestContracts/RuntimeHelper.cs]: check indentation

**Pull request merged**